scripts/cycled-yg-privacy.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+from future.utils import iteritems
+
+import random
+
+from jmbase import get_log, jmprint
+from jmbase.support import lookup_appdata_folder
+from jmclient import YieldGeneratorBasic, ygmain, jm_single
+
+# This is a maker for the purposes of generating a yield from held bitcoins
+# while maximising the difficulty of spying on blockchain activity.
+# This is primarily attempted by randomizing all aspects of orders
+# after transactions wherever possible.
+
+# YIELD GENERATOR SETTINGS ARE NOW IN YOUR joinmarket.cfg CONFIG FILE
+# (You can also use command line flags; see --help for this script).
+
+jlog = get_log()
+
+class YieldGeneratorPrivaCycler(YieldGeneratorBasic):
```

Comment:
See my non-commit comment for why I'm confused about "Cycle" in the name here, but also why "Priva" not "Private"?

Reply:
Oh I guess it's like a pun. Never mind :)